src/widgets/SimpleTable.js

```diff
@@ -123,6 +122,7 @@ export const SimpleTable = React.memo(
           ListHeaderComponent={renderHeaders}
           extraData={selectedRows}
           style={style}
+          {...flatListProps}
```

Comment:
means I can override `keyExtractor` as the vaccine history rows don't have an `id` - all the uses of `SimpleTable` seem to be OK